eyecite/models.py

```diff
@@ -50,14 +50,22 @@ class CitationBase:
 
     token: "Token"  # token this citation came from
     index: int  # index of _token in the token list
+    # span() overrides
+    span_start: Optional[int] = None
+    span_end: Optional[int] = None
 
     def matched_text(self):
         """Text that identified this citation, such as '1 U.S. 1' or 'Id.'"""
         return str(self.token)
 
     def span(self):
         """Start and stop offsets in source text for matched_text.()"""
-        return (self.token.start, self.token.end)
+        return (
+            self.span_start
+            if self.span_start is not None
+            else self.token.start,
+            self.span_end if self.span_end is not None else self.token.end,
+        )
```

Comment:
More elegant to say `return (self.span_start or self.token.start, self.span_end or self.token.end)`?

Reply:
That won't work for `self.span_start == 0` -- might work for `self.span_end` but probably clearer to keep it consistent.

Reply:
Good point.